gridengine/schedulers.py

```diff
@@ -152,7 +152,7 @@ def schedule(self, submission_host, job_queue, **resources):
     if not job_queue: return
 
     # update the keyword resources
-    resources = dict(self.resources.items() + resources.items())
+    resources.update(self.resources)
```

Comment:
This doesn't seem to work if resources is `None` (the default). 

I used instead:

```
if resources:
    resources = resources.update(self.resources)
else:
    resources = self.resources
```

cheers!